python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -225,6 +225,58 @@ interface {{ ethernet_interface.name }}
    dot1x authentication failure action traffic drop
 {%             endif %}
 {%         endif %}
+{%         if ethernet_interface.dot1x.aaa.unresponsive is arista.avd.defined %}
```

Comment:
if we can keep `ethernet_interface.dot1x.aaa.unresponsive` in a variable then this template will become much easier to read.

Reply:
Done.